docs/azure/sdk/resource-management.md

```diff
@@ -289,26 +322,26 @@ else
 // First, initialize the ArmClient and get the default subscription
 ArmClient armClient = new ArmClient(new DefaultAzureCredential());
 // Now we get a ResourceGroup collection for that subscription
-Subscription subscription = await armClient.GetDefaultSubscriptionAsync();
+SubscriptionResource subscription = await armClient.GetDefaultSubscriptionAsync();
 ResourceGroupCollection rgCollection = subscription.GetResourceGroups();
 
 // With the collection, we can create a new resource group with an specific name
 string rgName = "myRgName";
 Location location = Location.WestUS2;
 ResourceGroupData rgData = new ResourceGroupData(location);
-ResourceGroup resourceGroup = await rgCollection.CreateOrUpdateAsync(rgName, rgData);
+ResourceGroupResource resourceGroup = (await rgCollection.CreateOrUpdateAsync(rgName, rgData)).Value;
```

Comment:
This `.Value` is mandatory, because `ArmOperation<T>` does not have an implicit operator

Reply:
Are we planning to add implicit convert for ArmOperation like what we did for Azure.Response<T>? this inconsistency will confuse our customer

Reply:
> This `.Value` is mandatory, because `ArmOperation<T>` does not have an implicit operator

Are we planning to add implicit convert for ArmOperation like what we did for Azure.Response? this inconsistency will confuse our customer